.github/workflows/ci.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           --rm -v ${PWD}:/opt/conda-src
           -e TEST_SPLITS
           -e TEST_GROUP
-          ghcr.io/conda/conda-ci:master-linux-python${{ matrix.python-version }}
+          ghcr.io/conda/conda-ci:pr-11257-linux-python${{ matrix.python-version }}
```

Comment:
This'll need to be changed back to master, but is needed since the addition of /opt/conda/bin to PATH in the Dockerfile below made testing this impossible.